straxen/contexts.py

```diff
@@ -46,24 +46,25 @@
 
 def xenonnt_online(output_folder='./strax_data',
                    we_are_the_daq=False,
+                   _database_init=True,
                    **kwargs):
     """XENONnT online processing and analysis"""
     context_options = {
         **straxen.contexts.common_opts,
         **kwargs}
 
     st = strax.Context(
-        storage=[
-            straxen.RunDB(
-                readonly=not we_are_the_daq,
-                runid_field='number',
-                new_data_path=output_folder,
-                rucio_path='/dali/lgrandi/rucio/'),
-        ],
         config=straxen.contexts.xnt_common_config,
         **context_options)
     st.register([straxen.DAQReader, straxen.LEDCalibration])
 
+    st.storage = [
+        straxen.RunDB(
+            readonly=not we_are_the_daq,
+            runid_field='number',
+            new_data_path=output_folder,
+            rucio_path='/dali/lgrandi/rucio/'), ] \
```

Comment:
I very much dislike these "\\"-line breaks does not make the code very readable. Since st.storage is a simple list could not you do simply:

```
if _database_init:
    st.storage.append(....)
    # or st.storage.insert(0, ...) in case it shall be listed first
```

Reply:
I agree with you on the ''\'' in general. I don't feel strongly about it but the reason I think it's justifiable here is that for any non-oddly specific use cases (like this PR) you'd read it like you want it to be most of the time (you just init the rundb). 

I will change it though as I was also in doubt if this was really a good enough reason to write it like this.